tests/subsys/disk/stm32_sdmmc/prj.conf

```diff
@@ -0,0 +1,2 @@
+CONFIG_ZTEST=y
```

Comment:
tests are expected to provide normalized test verdict.
This should rather be moved to the sample section: samples/subsys